src/components/Sidebar/contentTree.js

```diff
@@ -9,6 +9,9 @@ import emoji from '../../utils/emoji';
 import { syncFunction } from '../../utils/utils';
 import { onMobile } from '../../styles/responsive';
 
+// Maximum Deep level of render Nav menu (Start with 0)
+const limitDeepLevel = 3;
```

Comment:
Should we pull this value to config.yml?

I suggest to use 

```yml
sidebar:
  limitDeepLevel: 3
```

Reply:
Nop, The existing state management is bad, I need to drill that prop down the tree. that is low priority.

Reply:
@mildronize  I agree. it id differ from what I work in layout or document level which is a few prob level.

BTW, you suggest to use React context or Redux for statement management?